stack.yaml

```diff
@@ -12,6 +13,22 @@ packages:
 
 extra-deps:
   - aeson-2.0.2.0
-  # https://github.com/yesodweb/shakespeare/pull/260
   # https://github.com/commercialhaskell/stackage/issues/6294
   - happy-1.20.0
+
+  # GHC 9.2 requirements
+  - attoparsec-0.14.2
+  - base-compat-0.12.1
+  - base-compat-batteries-0.12.1
+  - haskell-src-meta-0.8.8
+  # https://github.com/haskell-foundation/foundation/pull/555
+  - github: haskell-foundation/foundation
+    commit: 0bb195e1fea06d144dafc5af9a0ff79af0a5f4a0
+    subdirs:
+      - basement
+  # https://github.com/vincenthz/hs-memory/pull/87
+  - github: vincenthz/hs-memory
+    commit: 3cf661a8a9a8ac028df77daa88e8d65c55a3347a
+  # https://github.com/haskell-crypto/cryptonite/pull/354
+  - github: haskell-crypto/cryptonite
+    commit: 3b081e3ad027b0550fc87f171dffecbb20dedafe
```

Comment:
This is unfortunate. I wonder if we can drop these dependencies...

Reply:
It seems like commenting out `persistent-redis` and `persistent-mongoDB` from `stack.yaml` builds without these dependencies, but I can't figure out how those libraries use these libraries

Reply:
using `stack dot --external` and grepping the file, looks like `persistent-mongoDB` depends on `mongoDB` depends on `bson` depends on `cryptohash-md5` depends on `crpyothash` depends on `cryptonite`.

Reply:
Ah and `persistent-redis` depends on `hedis` which depends on `tls` which depends on `cryptonite`

Reply:
Oh, no, it's `mongoDB` -> `cryptohash` -> `cryptonite`. `cryptohash-md5` appears to be factored out to avoid that dependency entirely, but `mongoDB` appears to use `hmac` and `sha1` as well

Reply:
There is [cryptohash-sha1](https://hackage.haskell.org/package/cryptohash-sha1). And there is [crypto-api](https://hackage.haskell.org/package/crypto-api-0.13.3) which is maintained by @TomMD , I think.